src/rgw/rgw_asio_frontend.cc

```diff
@@ -533,15 +533,32 @@ int AsioFrontend::init()
 // Setting global timeout
   auto timeout = config.find("request_timeout_ms");
   if (timeout != config.end()) {
-    auto timeout_number = ceph::parse<uint64_t>(timeout->second.data());
```

Comment:
How did timeout->second's type change?

Reply:
this was a just a cosmetic cleanup. `timeout->second` is a `std::string`, and `ceph::parse()` takes `std::string_view`. `std::string_view` is happy to convert from `std::string`, so the extra conversion to `char* data()` wasn't necessary

Reply:
ah, ok; thanks for explaining